backend/tests/v5/unit/schemas/tickets/templates.constants.test.js

```diff
@@ -15,12 +15,18 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const { times, cloneDeep } = require('lodash');
 const { src } = require('../../../helper/path');
+const { generateRandomString } = require('../../../helper/services');
+const { statusTypes } = require('../../../../../src/v5/schemas/tickets/templates.constants');
```

Comment:
We should not be using relative path in the test files to reference source (i've pointed this out a few times before! 😆 )